### PR TITLE
[Bug] Fix backend addteammembers api  is Fixed

### LINKED ISSUE #872: [Bug] Fix backend addteammembers api
**Describe the bug**
There is problem with addTeamMembers api. Because it frontend is unable to add team members and throwing errors. fixing it is most important to fix frontend problems.

**To Reproduce**
1. open thunder client or postman
2. use auth token and try to addteam members.
3. it is not giving any response.

**Expected behavior**
when try to add team members with auth token. we should able to add the data.

**Have you read the [Code of Conduct](https://github.com/HITK-TECH-Community/Community-Website/blob/main/code_of_conduct.md)?**
yes

### PR DESCRIPTION
**Pull request by Hemu21**
## Issue that this pull request solves
There is problem with addTeamMembers api. Because it frontend is unable to add team members and throwing errors. fixing it is most important to fix frontend problems. By merging this pr. Backend is add team member endpoint is ready for Frontend development.

## Issue
[Issue Link](https://github.com/HITK-TECH-Community/Community-Website/issues/872) resolve #872 
 Closes: #872 

### Brief description of what is fixed or changed
I fixed the addteammembers endpoint file in backend. It has a function that function is wrong I corrected it.

## How to test it

- Way one

Check for the below thing in db. why because I added it.

`
{
description:"This is for checking the api",
full_name: "testing",
teams: ["Aman","anubhav"],
__v: 0,
_id: "664251826c3bb95cd441ebd9"
}
`

- way 2
try to add team members form this api.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by Hemu21** [comment]
@Kajol-Kumari can you please check this and merge it. then i will start working on #725

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Hemu21 what exactly have you fixed in this pr? There is an issue with `multer` upload and al i see in this pr is just comments and payload change

**Comment by Hemu21** [comment]
@Kajol-Kumari don't merge this pr.

**Comment by Hemu21** [comment]
@Kajol-Kumari don't merge this pr if you merge it again you will get error.

**Comment by Hemu21** [comment]
@Kajol-Kumari Now you can merge it.

**Comment by Hemu21** [comment]
@Kajol-Kumari can you please merge it. I will start my next issue.

File: backend/app/routes/faq/updateFaq.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/* eslint-disable no-shadow */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable prefer-const */
+/* eslint-disable consistent-return */
+/* eslint-disable no-unused-vars */
+/* eslint-disable prettier/prettier */
```
Comment (Kajol-Kumari) [inline backend/app/routes/faq/updateFaq.js:7]: can you please remove these un-necessary changes?

File: backend/app/routes/joinUs/deleteJoinUs.js [hunk 2]
```diff
@@ -1,3 +1,11 @@
+/* eslint-disable import/newline-after-import */
```
Comment (Kajol-Kumari) [inline backend/app/routes/joinUs/deleteJoinUs.js:1]: [neat]please remove these un-necessary changes.

File: backend/app/routes/teamMember/deleteTeamMember.js [hunk 3]
```diff
@@ -1,3 +1,11 @@
+/* eslint-disable import/newline-after-import */
```
Comment (Kajol-Kumari) [inline backend/app/routes/teamMember/deleteTeamMember.js:1]: [neat]please remove these un-necessary changes.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Hemu21  please remove those commented lines and it should be good to merge

File: backend/app/routes/faq/updateFaq.js (hunk 1 above)
Comment (Hemu21) [reply backend/app/routes/faq/updateFaq.js:7]: @Kajol-Kumari If i remove those 4 line while commiting the changes it askes to change the entire file.

File: backend/app/routes/teamMember/deleteTeamMember.js (hunk 3 above)
Comment (Hemu21) [reply backend/app/routes/teamMember/deleteTeamMember.js:1]: @Kajol-Kumari @Kajol-Kumari If i remove those 4 line while commiting the changes it askes to change the entire file. Changing the entire file may effect other function so i added that comments. They are unnecessary. but to commit changes they are necessary other wise we have to change all those files. I hope you understand.

Comment (Kajol-Kumari) [reply backend/app/routes/teamMember/deleteTeamMember.js:1]: the problem is these lines are disabling all the linting checks and so i can't allow it to be merged. can you please raise a new pr with just those 2 required change?

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**